<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QWidget" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_1"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_1" stretch="0,1,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="friendName" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(36, 200, 254);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>24</number>
        </property>
        <property name="topMargin">
         <number>12</number>
        </property>
        <property name="rightMargin">
         <number>12</number>
        </property>
        <property name="bottomMargin">
         <number>12</number>
        </property>
        <item>
         <widget class="QLabel" name="friendNameLabel">
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255)</string>
          </property>
          <property name="text">
           <string>冯玉彭</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="minimizeButton">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
    border-image: url(:/src/GUI/min.png);
    border-style: outset;
	border: none;
}
QPushButton:hover{
    image: url(:/src/GUI/min_hover.png);
    border-style: outset;
	border: none;
}
QPushButton:pressed{
    image: url(:/src/GUI/min_hover.png);
    border-style: outset;
	border: none;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
    border-image: url(:/src/GUI/close.png);
    border-style: outset;
}
QPushButton:hover{
    image: url(:/src/GUI/close_hover.png);
    border-style: outset;
}
QPushButton:pressed{
    image: url(:/src/GUI/close_hover.png);
    border-style: outset;
}
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">
QListWidget{background-color: rgb(255, 255, 255); color:rgb(51,51,51); border: none;outline:0px;}
QListWidget::Item{background-color: rgb(255, 255, 255);}
QListWidget::Item:hover{background-color: rgb(255, 255, 255); }
QListWidget::item:selected{
	background-color: rgb(255, 255, 255);
	color:black; 
    border: 1px solid  rgb(255, 255, 255);
}
QListWidget::item:selected:!active{border: 1px solid  rgb(255, 255, 255); background-color: rgb(255, 255, 255); color:rgb(51,51,51); } </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="msgSend" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QTextEdit" name="msgSendTextEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>120</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="sendBottom" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(240, 240, 240);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>9</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="font">
           <font>
            <family>Roboto</family>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="emojiButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#emojiButton{
 image: url(:/src/GUI/icon/emoji.png);
 border:none;
}
#emojiButton:hover{
 image: url(:/src/GUI/icon/emoji_hover.png);
 border:none;
}
#emojiButton:pressed{
 image: url(:/src/GUI/icon/emoji_press.png);
 border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="callButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#callButton{
 image: url(:/src/GUI/icon/call.png);
 border:none;
}
#callButton:hover{
 image: url(:/src/GUI/icon/call_hover.png);
 border:none;
}
#callButton:pressed{
 image: url(:/src/GUI/icon/call_press.png);
 border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="videoButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>18</width>
            <height>18</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#videoButton{
 image: url(:/src/GUI/icon/video.png);
 border:none;
}
#videoButton:hover{
 image: url(:/src/GUI/icon/video_hover.png);
 border:none;
}
#videoButton:pressed{
 image: url(:/src/GUI/icon/video_press.png);
 border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="voiceButton">
          <property name="minimumSize">
           <size>
            <width>23</width>
            <height>23</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#voiceButton{
 image: url(:/src/GUI/icon/voice.png);
 border:none;
}
#voiceButton:hover{
 image: url(:/src/GUI/icon/voice_hover.png);
 border:none;
}
#voiceButton:pressed{
 image: url(:/src/GUI/icon/voice_press.png);
 border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendPictureButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#sendPictureButton{
 image: url(:/src/GUI/icon/send_picture.png);
 border:none;
}
#sendPictureButton:hover{
 image: url(:/src/GUI/icon/send_picture_hover.png);
 border:none;
}
#sendPictureButton:pressed{
 image: url(:/src/GUI/icon/send_picture_press.png);
 border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendFileButton">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#sendFileButton{
 image: url(:/src/GUI/icon/send_file.png);
 border:none;
}
#sendFileButton:hover{
 image: url(:/src/GUI/icon/send_file_hover.png);
 border:none;
}
#sendFileButton:pressed{
 image: url(:/src/GUI/icon/send_file_press.png);
 border:none;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendPushButton">
          <property name="minimumSize">
           <size>
            <width>131</width>
            <height>54</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>131</width>
            <height>54</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Roboto</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	border-image: url(:/src/GUI/button.png);
	color:rgb(254, 254, 254);
}
QPushButton:hover{
	border-image: url(:/src/GUI/button_hover.png);
	color:rgb(254, 254, 254);
}
QPushButton:pressed{
	border-image: url(:/src/GUI/button_press.png);
	color:rgb(254, 254, 254);
}
</string>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
          <property name="icon">
           <iconset resource="src.qrc">
            <normaloff>:/src/GUI/send.png</normaloff>:/src/GUI/send.png</iconset>
          </property>
          <property name="shortcut">
           <string>Ctrl+Return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
